modules/oc-mirror-mirror-to-disk.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="oc-mirror-mirror-to-disk_{context}"]
+= Mirroring from mirror to disk
+
+You can use the oc-mirror plug-in to generate an image set and save the contents to disk. The generated image set can then be transferred to the disconnected environment and mirrored to the target registry.
+
+[IMPORTANT]
+====
+Depending on the configuration specified in the image set configuration file, using oc-mirror to mirror images might download several hundreds of gigabytes of data to disk.
+
+The initial image set to populate the mirror registry is often the largest. Subsequent executions of oc-mirror only download the changes since the previous execution, so the generated image set is often a smaller size.
+====
+
+You are required to specify a storage backend in the image set configuration file. This storage backend can be a local directory or a docker v2 registry. The oc-mirror plug-in stores metadata in this storage backend during image set creation.
+
```

Comment:
For the 'fully disconnected environment', we only could sepcify the storage backend  for local directory, can't be a docker registry.

Reply:
@zhouying7780 please see @afflom's comment below about the registry option still being available to use here.